tiny_dnn/util/util.h

```diff
@@ -358,10 +358,11 @@ inline size_t pool_out_length(size_t in_length,
     throw nn_error("Not recognized pad_type.");
   }
 
-  if ( ceil_mode ) {
-    return static_cast<int>(ceil(static_cast<float>((output_length + stride - 1)) / stride));
+  float tmp = static_cast<float>((output_length + stride - 1)) / stride;
+  if (ceil_mode) {
+    return static_cast<int>(ceil(tmp));
   } else {
-    return static_cast<int>(floor(static_cast<float>((output_length + stride - 1)) / stride));
+    return static_cast<int>(floor(tmp));
```

Comment:
Thank you for updating the code.
I've just thought there's another way of writing this code.

```cpp
  auto tmp = static_cast<float_t>(output_length + stride - 1) / stride;
  return static_cast<int>(ceil_mode ? ceil(tmp) : floor(tmp));
```

Reply:
Cool. Very simple.
I'll update it.